sdk/attestation/azure-security-attestation/vcpkg/Config.cmake.in

```diff
@@ -4,7 +4,7 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
-find_dependency(azure-core-cpp "1.5.0")
+find_dependency(azure-core-cpp "1.7.0")
```

Comment:
@antkmsft am I right to assume the version specified here in `Config.cmake.in` needs to match the minimum version we require in `vcpkg.json`?

Reply:
Yes, but in case of betas, we can't put `-beta.X` for `find_dependency()` - that one only supports only numeric versions, IIRC. Now that everyting is released, it is a good change to make. Thank you!